environment/src/main/java/com/sequenceiq/environment/platformresource/v1/CredentialPlatformResourceController.java

```diff
@@ -59,23 +57,67 @@
 import com.sequenceiq.environment.api.v1.platformresource.model.TagSpecificationsResponse;
 import com.sequenceiq.environment.platformresource.PlatformParameterService;
 import com.sequenceiq.environment.platformresource.PlatformResourceRequest;
+import com.sequenceiq.environment.platformresource.v1.converter.CloudAccessConfigsToPlatformAccessConfigsV1ResponseConverter;
+import com.sequenceiq.environment.platformresource.v1.converter.CloudEncryptionKeysToPlatformEncryptionKeysV1ResponseConverter;
+import com.sequenceiq.environment.platformresource.v1.converter.CloudGatewayssToPlatformGatewaysV1ResponseConverter;
+import com.sequenceiq.environment.platformresource.v1.converter.CloudIpPoolsToPlatformIpPoolsV1ResponseConverter;
+import com.sequenceiq.environment.platformresource.v1.converter.CloudNetworksToPlatformNetworksV1ResponseConverter;
+import com.sequenceiq.environment.platformresource.v1.converter.CloudNoSqlTablesToPlatformNoSqlTablesV1ResponseConverter;
+import com.sequenceiq.environment.platformresource.v1.converter.CloudSecurityGroupsToPlatformSecurityGroupsV1ResponseConverter;
+import com.sequenceiq.environment.platformresource.v1.converter.CloudSshKeysToPlatformSshKeysV1ResponseConverter;
+import com.sequenceiq.environment.platformresource.v1.converter.CloudVmTypesToPlatformVmTypesV1ResponseConverter;
+import com.sequenceiq.environment.platformresource.v1.converter.PlatformDisksToPlatformDisksV1ResponseConverter;
+import com.sequenceiq.environment.platformresource.v1.converter.PlatformRegionsToRegionV1ResponseConverter;
+import com.sequenceiq.environment.platformresource.v1.converter.TagSpecificationsToTagSpecificationsV1ResponseConverter;
 
 @Controller
 @Transactional(TxType.NEVER)
 public class CredentialPlatformResourceController implements CredentialPlatformResourceEndpoint {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CredentialPlatformResourceController.class);
 
-    @Inject
-    @Named("conversionService")
-    private ConversionService convertersionService;
-
     @Inject
     private PlatformParameterService platformParameterService;
 
     @Inject
     private CommonPermissionCheckingUtils commonPermissionCheckingUtils;
 
+    @Inject
```

Comment:
What was changed that now we can't use the conversion service but we have to import all converters separately?

Reply:
and you need to invoke them directly.